### ISSUE TITLE
Missing DLL files?

### ISSUE THREAD

**Issue by nathantgray** [SunPower/SolarUtils#18]
After pip installing SolarUtils on Windows 10 and trying to run the example code in the readme file I get this error. Looking at the core.py code, it appears that the program is looking for solposAM.dll in the same folder (solar_utils), but they are not there. When I install it on my mac there is no problem and the libsolposAM.dylib is in the solar_utils folder.

The following is from the terminal. (Pardon the unintentional formatting.)

```C:\ProgramData\Anaconda3\Lib\site-packages\solar_utils\src>ipython
Python 3.6.5 |Anaconda, Inc.| (default, Mar 29 2018, 13:32:41) [MSC v.1900 64 bit (AMD64)]
Type 'copyright', 'credits' or 'license' for more information
IPython 6.4.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from solar_utils import *

In [2]: import pandas as pd

In [3]: location = [35.56836, -119.2022, -8.0]

In [4]: datetime = [2013, 6, 5, 12, 31, 0]

In [5]: weather = [1015.62055, 40.0]

In [6]: (angles, airmass) = solposAM(location, datetime, weather)
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
<ipython-input-6-cb97c5201d9e> in <module>()
----> 1 (angles, airmass) = solposAM(location, datetime, weather)

C:\ProgramData\Anaconda3\lib\site-packages\solar_utils\core.py in solposAM(location, datetime, weather)

    164     """
    165     # load the DLL
--> 166     solposAM_dll = ctypes.cdll.LoadLibrary(SOLPOSAMDLL)
    167     _solposAM = solposAM_dll.solposAM
    168     # cast Python types as ctypes

C:\ProgramData\Anaconda3\lib\ctypes\__init__.py in LoadLibrary(self, name)
    424
    425     def LoadLibrary(self, name):
--> 426         return self._dlltype(name)
    427
    428 cdll = LibraryLoader(CDLL)

C:\ProgramData\Anaconda3\lib\ctypes\__init__.py in __init__(self, name, mode, handle, use_errno, use_la
st_error)
    346
    347         if handle is None:
--> 348             self._handle = _dlopen(self._name, mode)
    349         else:
    350             self._handle = handle

OSError: [WinError 126] The specified module could not be found```

**Comment by anomam** [comment]
@mikofski do you have any insight here by any chance?

**Comment by mikofski** [comment]
It looks like there's something wrong with the 3.6 wheel on pypi, can you please use the 3.7 wheel instead? Sorry

**Comment by nathantgray** [comment]
Thank you, that fixed it.

**Comment by mikofski** [comment]
Notes for the future:

1. I've uploaded a FIXED version of the Python-3.6 wheel to PyPI and GitHub releases page. 
1. evidently calling `python setup.py sdist bdist_wheel` ends up creating a wheel with the contents of a source distribution, so I recommend separating these commands in the future, and execute `python setup.py bdist_wheel` separately
2. The long term solution to this, I believe is to use AppVeyor to build and deploy wheels for Windows automatically.
